ui/src/translations/de.ts

```diff
@@ -66,11 +66,11 @@ export const de = {
     unban_from_site: 'Von der Seite entbannen',
     banned: 'gesperrt',
     save: 'speichern',
-    unsave: 'unsave',
+    unsave: 'rückgängig speichern',
```

Comment:
Das macht so keinen Sinn auf Deutsch. Besser "rückgängig machen" wie ich vorgeschlagen hab.

Reply:
richtert!

Reply:
Sorry, nur "rückgängig machen", ohne "speichern" an der Stelle. Und "richtert" ist kein Wort :p

Sonst sieht es aber gut aus 👍

Reply:
Obwohl, ohne das Wort "speichern" weiß man dann wahrscheinlich gar nicht, um was es geht. Vllt besser mit "speichern rückgängig machen".

Reply:
I mean, how would they know that's the button for unsaving and not undoing something else then?

Reply:
Does it still need changing?

Reply:
You're right, thats what I mentioned in my last comment. I think "speichern rückgängig machen" is good, but the way you put it now just sounds wrong.

Reply:
Just checked, Reddit uses "Nicht speichern".

Reply:
Should I just change it to that?

Reply:
Oh yeah smart to check that. Its not the exact same meaning, but its definitely shorter and easier so we should probably go with that.

Reply:
Changed!